filters/style_manager.py

```diff
@@ -109,6 +109,12 @@ def responseComplete(self):
                 qgis_layer = qgis_layer[0]
 
             # NAME
+            # Check default name should be `default` to follow GetCapabilities
+            # response
+            style_manager = qgis_layer.styleManager()
+            style_manager.renameStyle('', 'default')
+            project.write()
```

Comment:
What I did to rename the style from empty to `default`.
Just a temporary hack for me until we figured out what the problem.